.github/workflows/drive.yml

```
@@ -94,13 +88,26 @@ jobs:
             let resultsTableDivider = `| --- | --- |`;
             // add everything to the resultsTable
             resultsTable = resultsTableHeader + '\n' + resultsTableDivider + '\n' + resultsTable.join('\n');
+            return resultsTable;
+          result-encoding: string
+      - name: Print simulation results
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2028:info:1:6: echo may not expand escape sequences. Use printf

(shellcheck)